vm/systemSmartContracts/staking.go

```diff
@@ -1543,32 +1545,31 @@ func (s *stakingSC) resetLastUnJailedFromQueue(args *vmcommon.ContractCallInput)
 
 func (s *stakingSC) cleanAdditionalQueueNotEnoughFunds(
```

Comment:
cleanAdditionalQueueNotEnoughFundsReturningUnstaked is a better name now

Reply:
that is too long.